examples/md-flexible/tests/TimeDiscretizationTest.cpp

```diff
@@ -126,8 +125,12 @@ void TimeDiscretizationTest::Pos_and_Velo_Test(
 #endif
     for (auto iter = autopas.begin(); iter.isValid(); ++iter) {
       EXPECT_EQ(iter->getID(), i);
-      EXPECT_EQ(iter->getV(), nextVelocity(oldVelocityValues.at(i), velForces.at(i), forces.at(i), particleD));
-      EXPECT_EQ(iter->getR(), nextPosition(oldPositionValues.at(i), forces.at(i), oldVelocityValues.at(i), particleD));
+      auto expectedV = nextVelocity(oldVelocityValues.at(i), velForces.at(i), forces.at(i), deltaT);
+      auto expectedR = nextPosition(oldPositionValues.at(i), forces.at(i), oldVelocityValues.at(i), deltaT);
+      for (int dim = 0; dim < 3; ++dim) {
+        EXPECT_NEAR(iter->getV()[dim], expectedV[dim], 1e-13);
+        EXPECT_NEAR(iter->getR()[dim], expectedR[dim], 1e-13);
+      }
       i++;
```

Comment:
TBH this test looks sketchy. I will rework it in a different branch but this version here at least passes...

Reply:
did this not pass before this change?

Reply:
failed for me once while testing because it compared doubles via ==